index.js

```diff
@@ -1,2 +1,15 @@
 function addingEventListener() {
+
+
+const input = document.getElementById('button');
+
+function clickAlert() {
+  alert('I was clicked!');
 }
+
+input.addEventListener('click', clickAlert);
+
+
+}
+
+addingEventListener()
```

Comment:
This spacing is a little all over the place. Try to keep it a bit more consistent next time, thanks!